app/utils/asset.js

```diff
@@ -120,16 +120,18 @@ export default class AssetUtil {
     const descendantsList = [];
     // check if the asset uri has a descendant root uri in it
     if (assetUri && rootUri && assetUri.includes(rootUri) && assetUri.indexOf(rootUri) === 0) {
+      // because for windows the path seperator is '\'
+      const sep = process.platform === 'win32' ? '\\' : '/';
```

Comment:
I think you can use the built-in `path.sep` instead of deriving it yourself (https://nodejs.org/docs/latest/api/path.html#pathsep)